libs/sdk-ui-dashboard/src/model/events/filters.ts

```diff
@@ -16,12 +16,14 @@ export interface DashboardDateFilterSelectionChanged extends IDashboardEvent {
     readonly type: "GDC.DASH/EVT.FILTER_CONTEXT.DATE_FILTER.SELECTION_CHANGED";
     readonly payload: {
         readonly filter: IDashboardDateFilter;
+        readonly dateFilterOptionLocalId: string | undefined;
     };
 }
 
 export function dateFilterChanged(
     ctx: DashboardContext,
     filter: IDashboardDateFilter,
+    dateFilterOptionLocalId: string | undefined,
```

Comment:
Nitpick: can be `dateFilterOptionLocalId?: string`

Reply:
Fixed